tests/e2e/instrumentation-dotnet-multicontainer/01-install-app.yaml

```diff
@@ -18,6 +18,6 @@ spec:
     spec:
       containers:
       - name: myapp
-        image: docker.io/avadhutp123/aspnetapp:latest # source code of the application: https://github.com/dotnet/dotnet-docker/tree/main/samples/aspnetapp
+        image: docker.io/avadhutp123/aspnetapp@sha256:d2e8d3415f6f12efae0369aa0a9777a58583841fb133f33e10a73adb9fb392da # source code of the application: https://github.com/dotnet/dotnet-docker/tree/main/samples/aspnetapp
```

Comment:
@pavolloffay do you know why we're not using the dotnet images from microsoft? i.e. `mcr.microsoft.com/dotnet/samples:aspnetapp`?

Reply:
@jaronoff97 The reason is described in https://github.com/open-telemetry/opentelemetry-operator/pull/1151